specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/Servers.json

```diff
@@ -327,6 +382,221 @@
     }
   },
   "definitions": {
+    "ImportNewDatabaseDefinition": {
+      "description": "Contains the information necessary to perform import operation for new database.",
+      "required": [
+        "storageKeyType",
+        "storageKey",
+        "storageUri",
+        "administratorLogin",
+        "administratorLoginPassword"
+      ],
+      "type": "object",
+      "properties": {
+        "databaseName": {
+          "description": "Name of the import database.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "edition": {
+          "description": "Edition of the import database.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "serviceObjectiveName": {
+          "description": "Service level objective name of the import database.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "maxSizeBytes": {
+          "description": "Max size in bytes for the import database.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "storageKeyType": {
+          "description": "Storage key type.",
+          "enum": [
+            "SharedAccessKey",
+            "StorageAccessKey"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "StorageKeyType",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "storageKey": {
```

Comment:
Is this a secret?

Make sure to add: 

"x-ms-secret": true

Reply:
Same for your 2020-08-01-preview api

Reply:
we would try our best to catch up with the change to the latest version: 2020-08-01-preview.

Reply:
Thanks! Not sure if this something you can add but probably for future PRs this info can be added in the PR description.